deploy/csv/redhat/1.4.0/manifests/percona-xtradb-cluster-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -97,7 +97,6 @@ spec:
       root: cm9vdF9wYXNzd29yZA==
       xtrabackup: YmFja3VwX3Bhc3N3b3Jk
       monitor: bW9uaXRvcg==
-      clustercheck: Y2x1c3RlcmNoZWNrcGFzc3dvcmQ=
```

Comment:
@inelpandzic, why do we need to remove it for 1.4.0, 1.5.0, 1.6.0, 1.7.0 and 1.8.0?

Reply:
Yes, good point, we probably don't.

Reply:
Done [e61172c](https://github.com/percona/percona-xtradb-cluster-operator/pull/1542/commits/e61172cc11c48338e4870775c16bbc0a797b2aa3)